.luacheckrc

```diff
@@ -23,13 +23,14 @@ read_globals = {
     "mouse",
     "mousegrabber",
     "root",
-    "screen",
     "selection",
     "tag",
     "window",
 }
+
 -- May not be read-only due to client.focus
 globals = {
+    "screen",
```

Comment:
Could add to the commit message and the comment before this array the reason why `screen` can not be read-only?